packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -89,23 +266,21 @@ const ComboBox = React.forwardRef((props, ref) => {
     light,
     onChange,
     onInputChange,
-    onToggleClick, // eslint-disable-line no-unused-vars
+    onToggleClick, 
```

Comment:
Where is this being used now that we can remove the eslint line disable?

Reply:
It is being used here https://github.com/carbon-design-system/carbon/blob/e3823aa2e0b14022df8ba86b91edf61ad6d4948e/packages/react/src/components/ComboBox/ComboBox.tsx#L363